src/better-sandwiches/Makefile

```diff
@@ -1,2 +0,0 @@
-serve:
-	app=better-sandwiches make -C ../.. serve
```

Comment:
Why are the submakes not necessary anymore?

Reply:
The project-specific Makefile is no longer necessary because of [Vite's multi-page app feature](https://vitejs.dev/guide/build#multi-page-app). They've essentially been replaced by project-specific index.html files.